src/tests/Utils.test.js

```diff
@@ -1,27 +1,32 @@
 import { convertPostcode } from "../utils/convertPostcode";
 import { getData } from "../utils/getData";
 
-describe.only("Convert postcode to lon and lat", () => {
+describe.only("ConvertPostcodeConvert postcode to lon and lat", () => {
+  const expected = {
+    longitude: -2.008298,
+    latitude: 50.712412
+  };
+  const initialPostcode = "BH15 4DH";
   test("ConvertPostcode returns correct longitude and latitude ", () => {
-    const expected = {
-      longitude: -2.008298,
-      latitude: 50.712412
-    };
-    const initialPostcode = "BH15 4DH";
     return convertPostcode(initialPostcode).then(actual => {
       expect(actual).toEqual(expected);
+      expect(initialPostcode).toEqual("BH15 4DH");
```

Comment:
This will always pass, I removed it on the previous time, now sure why you have added it again